sdk/dapp-kit/src/index.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+export * from './hooks/index.js';
```

Comment:
wondering if it makes sense to start out with a flat tree, i.e. exports just live under `src/` - this would make more sense if we're not going to be adding much to the `components` directory

Reply:
sounds good to me, I'll update this